tests/virt_autotest/login_console.pm

```diff
@@ -95,8 +95,15 @@ sub login_to_console {
 }
 
 sub run {
+    if (check_var('ARCH', 's390x')) {
+        #Switch to s390x lpar console
+        reset_consoles;
+        my $svirt               = select_console('svirt', await_console => 0);
+    }
+    else {
     my $self = shift;
     $self->login_to_console;
+    }
```

Comment:
You could move your new code into `login_to_console` as basically you are still doing the same from the name.

Reply:
Had made code changed as your mentioned, after local verification run, will update this PR,ASAP.

Reply:
Move the new code into login_to_console as basically, updated this PR and
Verification Run:
http://10.67.17.251/tests/90